### DISCUSSION TITLE
[Vue warn]: Property "item" was accessed during render but is not defined on instance.

### DISCUSSION THREAD

**Discussion by gaguerrero** [quasarframework/quasar#9928] [General - Components / Directives / etc]
[Vue warn]: Property "item" was accessed during render but is not defined on instance.

I am getting that error after migrating to quasar 2. Can someone help me with an example?
Here is my source code:

[saldos.vue.txt](https://github.com/quasarframework/quasar/files/6757265/saldos.vue.txt)

**Comment by metalsadman** [comment]
like the error suggests, you are using a variable not defined in script part of your code. you probably have to check first if `item` is undefined or not before using it.

**Comment by ontwikkelfabriek** [comment]
@gaguerrero 

> I am getting that error after migrating to quasar 2. 

That's because of a **breaking** change in Vue3:
 _If used on the **same** element, **v-if** will have **higher** precedence than **v-for**_
 
 See:
https://v3.vuejs.org/guide/migration/v-if-v-for.html

**Reply by gaguerrero** [reply]
Thank you very much for your help, it helped me a lot!
